### PR TITLE
mysql: add Chinese translation

### PR DESCRIPTION
**Pull request by witt-bit**
- [✔] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [✔] The page(s) have at most 8 examples.
- [✔] The page description(s) have links to documentation or a homepage.
- [✔] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [✔] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [✔] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Review by spageektti — COMMENTED** [review]
Hi. Could you please translate the placeholders too?

**Comment by witt-bit** [comment]
> Hi. Could you please translate the placeholders too?

Updated

File: pages.zh/common/mysql.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+`mysql -u {{用户名}} --password {{数据库名}}`
+
+- 连接到另一台主机上的数据库：
+
+`mysql -h {{数据库地址}} {{数据库名}}`
+
+- 通过Unix套接字文件连接到数据库：
+
+`mysql --socket {{socket.sock文件路径}}`
```
Comment (blueskyson) [inline pages.zh/common/mysql.md:20]:
```suggestion
`mysql --socket {{路径/到/socket.sock}}`
```

File: pages.zh/common/mysql.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+`mysql -e "source {{SQL脚本}}" {{数据库名}}`
+
+- 从`mysqldump`创建的备份文件中恢复单个数据库（系统将提示用户输入密码）：
+
+`mysql --user {{用户名}} --password {{数据库名}} < {{备份SQL文件路径}}`
+
+- 从备份中恢复所有数据库（系统将提示用户输入密码）：
+
+`mysql --user {{用户名}} --password < {{备份SQL文件路径}}`
```
Comment (blueskyson) [inline pages.zh/common/mysql.md:32]:
```suggestion
`mysql --user {{用户名}} --password < {{路径/到/备份文件.sql}}`
```

File: pages.zh/common/mysql.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+`mysql --socket {{socket.sock文件路径}}`
+
+- 执行脚本文件中的SQL语句：
+
+`mysql -e "source {{SQL脚本}}" {{数据库名}}`
+
+- 从`mysqldump`创建的备份文件中恢复单个数据库（系统将提示用户输入密码）：
+
+`mysql --user {{用户名}} --password {{数据库名}} < {{备份SQL文件路径}}`
```
Comment (blueskyson) [inline pages.zh/common/mysql.md:28]:
```suggestion
`mysql --user {{用户名}} --password {{数据库名}} < {{路径/到/备份文件.sql}}`
```

File: pages.zh/common/mysql.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
+# mysql
+
+> MySQL命令行工具.
+> 更多信息: <https://www.mysql.com/cn/>.
```
Comment (blueskyson) [inline pages.zh/common/mysql.md:4]:
```suggestion
> MySQL 命令行工具。
> 更多信息：<https://www.mysql.com/cn/>.
```

File: pages.zh/common/mysql.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+`mysql -h {{数据库地址}} {{数据库名}}`
+
+- 通过Unix套接字文件连接到数据库：
+
+`mysql --socket {{socket.sock文件路径}}`
+
+- 执行脚本文件中的SQL语句：
+
+`mysql -e "source {{SQL脚本}}" {{数据库名}}`
```
Comment (blueskyson) [inline pages.zh/common/mysql.md:24]:
```suggestion
`mysql -e "source {{脚本.sql}}" {{数据库名}}`
```

**Review by blueskyson — COMMENTED** [review]
Thanks for contribution. I left some suggestions.

File: pages.zh/common/mysql.md (hunk 2 above)
Comment (witt-bit) [reply pages.zh/common/mysql.md:32]: OK, 这翻译的跟人机似的

**Review by blueskyson — APPROVED** [review]

**Review by spageektti — APPROVED** [review]
Thanks!

**Pull request merged**